doc/release_notes/broker19.04.rst

```diff
@@ -0,0 +1,12 @@
+=======================
+Centreon Broker 19.04.0
+=======================
+
```

Comment:
Can you make the same kind of release note 
https://documentation.centreon.com/docs/centreon-open-tickets/en/latest/release_notes/centreon-open-tickets-19.04/index.html